src/coreclr/jit/instrsarm64sve.h

```diff
@@ -239,6 +239,43 @@ INST7(ld1sw,             "ld1sw",                 0,                       IF_SV
     // LD1SW   {<Zt>.D }, <Pg>/Z, [<Xn|SP>, <Zm>.D]                                      SVE_IU_4B_B         11000101010mmmmm 100gggnnnnnttttt     C540 8000   
     // LD1SW   {<Zt>.D }, <Pg>/Z, [<Zn>.D{, #<imm>}]                                     SVE_IV_3A           11000101001iiiii 100gggnnnnnttttt     C520 8000   
 
+//    enum               name                     info                                              SVE_HP_3B        SVE_HP_3B_H      SVE_HP_3B_I      SVE_HP_3B_J  SVE_HP_3B_K   SVE_HP_3B_L    SVE_HP_3B_M 
+INST7(fcvtzs,            "fcvtzs",                0,                       IF_SVE_7B,               0x659CA000,      0x65DCA000,      0x65D8A000,      0x65DEA000,  0x655AA000,   0x655CA000,    0x655EA000       )
```

Comment:
The H variant was missing from all these encodings. The changes here look fine to me, but I'll let @kunalspathak comment